src/views/SelectProduct.vue

```diff
@@ -17,73 +17,84 @@
 
     <ion-content>
       <div class="find">
-        <section class="search">
-          <ion-searchbar :placeholder="$t('Search products')" />
-        </section>
 
         <aside class="filters desktop-only">
           <ion-list>
-            <ion-list-header><h3>{{ $t("Include") }}</h3></ion-list-header>
+            <ion-item lines="inset">
+              <ion-label position="fixed">{{ $t("Threshold") }}</ion-label>
+              <ion-input type="text" :placeholder="$t('global threshold')"/>
+            </ion-item>
+            <ion-list-header>
+              <ion-label>{{ $t("Include") }}</ion-label>
+              <ion-button fill="clear" color="warning">RESET</ion-button>
+            </ion-list-header>
             <ion-card>
               <ion-toolbar>
                 <ion-title>{{ $t("Tags") }}</ion-title>
+                <ion-button fill="clear" slot="end">
+                 <ion-label>Add</ion-label>
```

Comment:
@disha1202 if you made previous button label all caps, then this should also be all caps. Code patterns, even when wrong, should be consistent. Inconsistent code just as bad as broken code.